libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -804,6 +804,61 @@ button.gd-button-primary {
     }
 }
 
+.gd-upsell-button {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 14px;
+    width: 93px;
+    height: 27px;
+    text-align: center;
+    color: #ffffff;
+    margin: auto 7px;
+    background: #8351a4;
+    padding: 0;
+
+    .gd-upgrade-button-text {
+        margin-right: 11px;
+        font-size: 12px;
+        font-family: $gd-font-primary;
+    }
+
+    .gd-icon-star {
+        background: url("~@gooddata/sdk-ui-kit/esm/assets/star.svg") no-repeat;
+        height: 13px;
+        width: 14px;
+        margin: auto auto 6.5px 11px;
+    }
+
+    &:hover {
+        background: linear-gradient(0deg, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)), #8351a4;
+        border: 1px solid rgba(0, 0, 0, 0.1);
+        box-sizing: border-box;
+        box-shadow: 0 1px 2px rgba(0, 0, 0, 0.15), inset 0px -2px 0px rgba(0, 0, 0, 0.15);
+        border-radius: 3px;
+        color: #ffffff;
+    }
+
+    &:focus {
+        background: #8351a4;
+        border: 1px solid rgba(0, 0, 0, 0.2);
+        box-sizing: border-box;
+        box-shadow: 0px 0px 5px 2px rgba(131, 81, 164, 0.6), 0px 1px 2px rgba(0, 0, 0, 0.15),
+            inset 0px -2px 0px rgba(0, 0, 0, 0.15);
+        border-radius: 3px;
+        color: #ffffff;
+    }
+
+    &:active {
+        background: linear-gradient(180deg, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.1) 100%), #8351a4;
+        border: 1px solid rgba(0, 0, 0, 0.1);
+        box-sizing: border-box;
+        box-shadow: inset 0px 2px 0px rgba(0, 0, 0, 0.15);
+        border-radius: 3px;
+        color: #ffffff;
```

Comment:
I think all states for this button has the same `border-radius`, `color`, and `box-sizing`. Can you double check the mockup, I think we can move them to the normal state.

Reply:
I moved border-radius, color into normal state, about box-sizing I will keep it as it is due to it doens't have in normal state

Reply:
Cool 😎